db/migrate/20181214143356_create_users.rb

```diff
@@ -0,0 +1,21 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users, id: :uuid do |t|
+      t.string :username, null: false
+      t.string :type
```

Comment:
Do we now need to remove the `providers` table?
If so, it might be safer to do it in a subsequent PR.

Ideally, we would migrate the records from the `providers` table to this new `users` table but maybe we don't need to worry about that as we are not live.